common/datasets/switchboard.py

```diff
@@ -0,0 +1,198 @@
+from i6_core.datasets.switchboard import *
+from i6_core.lexicon.conversion import LexiconFromTextFileJob
+from i6_core.lexicon.modification import WriteLexiconJob, MergeLexiconJob
+from i6_core.lib import lexicon
+
+
+def get_speakers_list(subdir_prefix=""):
+    """
+    Returns speakers list
+
+    :param str subdir_prefix: alias name prefix
+    :rtype: tk.Path
```

Comment:
if the `rtype` is only `Path`, please also specify with `:return:` in a line directly above what kind of file is returned.